spec/requests/groups_controller_spec.rb

```diff
@@ -166,6 +166,14 @@
 
         expect(Group.last.creator).to eq(editor_user)
       end
+
+      it "gives the creator the group admin role" do
+        login_as_editor_user
+
+        post groups_url, params: { group: valid_attributes }
+
+        expect(Membership.last.role).to eq("group_admin")
```

Comment:
I'm probably being a little overcautious, but it might be worth tightening up this test a little?

```suggestion
        expect(Membership.last).to have_attributes user: editor_user, role: :group_admin
```

Reply:
Good suggestion, have implemented!